### ISSUE TITLE
Could not execute entrypoint stage 'client' due to errors!

### ISSUE THREAD

**Issue by Jarfeh** [kyrptonaught/linkedstorage#5]
When I attempt to launch 1.15.2 with fabric installed, the game window never opens, instead, the launcher pops back up with the error: java.lang.RuntimeException: Could not execute entrypoint stage 'client' due to errors!

[crash-2020-02-28_20.20.42-client.txt](https://github.com/kyrptonaught/linkedstorage/files/4269908/crash-2020-02-28_20.20.42-client.txt)
I can launch fine without the mod.

**Comment by kyrptonaught** [comment]
Does it work without optifine?

**Comment by su-minus** [comment]
I had the same issue, and running the game without Optifine/Optifabric works.
Versions used:
- Linked Storage: 1.1.2 (I also tried 1.1.1 and it crashed too, so I assume all versions have the same problem)
- Optifabric: 1.0.0 beta 4
- Optifine: 1.15.2 HD U G1 pre9

[Here is my crash report](https://github.com/kyrptonaught/linkedstorage/files/4283127/crash-2020-03-01_23.48.41-client.txt).

I did try to find what could have caused the NPE on Optifine's side, and I've written all I could find [here](https://pastebin.com/YuhgAqDH).
